### DISCUSSION TITLE
reading the hardware "licznik" locally

### DISCUSSION THREAD

**Discussion by ivdimitro** [PiotrMachowski/Home-Assistant-custom-components-Tauron-AMIplus#78] [Q&A] [answered]
Hi, a bit off topic but do you know what radio module is needed to read the "licznik" directly and not via the Tauron API? Mine is model "83332" and supposed to support Wireless M-Bus. 
Do you what and from where I can by this wireless M-Bus dongle for home assistant?

**Comment by PiotrMachowski** [comment]
The best I can do is to point you to the documentation of [wmbusmeters](https://github.com/weetmuts/wmbusmeters) 

---

For such questions please use [discussions](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Tauron-AMIplus/discussions) instead of issues

**Reply by ivdimitro** [reply, marked as answer]
Thanks for the suggestion. With Cheap DVB adapter and HA add on [https://github.com/wmbusmeters/wmbusmeters-ha-addon](https://github.com/wmbusmeters/wmbusmeters-ha-addon) the power consumption reading is a breeze. 
Really appreciate your work but this solution seems much easier and less prone to the constant API changes. I guess there will be users for both.

**Reply by Rettiwer** [reply]
Can you give a model of the adapter?
